src/main/commandServer/httpCommandServer.ts

```diff
@@ -187,10 +187,14 @@ export class HttpCommandServer {
     let result = '';
     const [data, payloadError] = await serverHelper.getRequestBody(request, MAX_REQUEST_BODY_LENGTH);
```

Comment:
Shouldn't this call now also use the new 3-value return form of `getRequestBody`?